plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -109,23 +115,23 @@
 import static java.lang.String.join;
 
 public class ClickHouseClient
-        extends BaseJdbcClient
-{
+        extends BaseJdbcClient {
```

Comment:
Can you please reformat the code with the project's code style?
see https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md#code-style for how to configure it

Reply:
Ok, I configure it